site/javadocs/allclasses-frame.html

```diff
@@ -32,12 +32,24 @@
     <h1 class="bar">All Classes</h1>
     <div class="indexContainer">
         <ul>
-        <li><a href="pluggable.scm/PropertiesSCMProviderDataStore.html" title="class in PropertiesSCMProviderDataStore" target="classFrame">PropertiesSCMProviderDataStore</a></li>
-<li><a href="pluggable.scm/SCMProvider.html" title="interface in SCMProvider" target="classFrame"><I>SCMProvider</I></a></li>
-<li><a href="pluggable.scm/SCMProviderDataStore.html" title="interface in SCMProviderDataStore" target="classFrame"><I>SCMProviderDataStore</I></a></li>
-<li><a href="pluggable.scm/SCMProviderFactory.html" title="interface in SCMProviderFactory" target="classFrame"><I>SCMProviderFactory</I></a></li>
-<li><a href="pluggable.scm/SCMProviderHandler.html" title="class in SCMProviderHandler" target="classFrame">SCMProviderHandler</a></li>
-<li><a href="pluggable.scm/SCMProviderInfo.html" title="annotation type in SCMProviderInfo" target="classFrame">SCMProviderInfo</a></li>
+        <li><a href="pluggable/scm/bitbucket/BitbucketRequestUtil.html" title="class in pluggable.scm.bitbucket" target="classFrame">BitbucketRequestUtil</a></li>
+<li><a href="pluggable/scm/bitbucket/BitbucketSCMProtocol.html" title="enum in pluggable.scm.bitbucket" target="classFrame">BitbucketSCMProtocol</a></li>
+<li><a href="pluggable/scm/bitbucket/BitbucketSCMProvider.html" title="class in pluggable.scm.bitbucket" target="classFrame">BitbucketSCMProvider</a></li>
+<li><a href="pluggable/scm/bitbucket/BitbucketSCMProviderFactory.html" title="class in pluggable.scm.bitbucket" target="classFrame">BitbucketSCMProviderFactory</a></li>
+<li><a href="pluggable/configuration/EnvVarProperty.html" title="class in pluggable.configuration" target="classFrame">EnvVarProperty</a></li>
+<li><a href="pluggable/scm/helpers/ExecuteShellCommand.html" title="class in pluggable.scm.helpers" target="classFrame">ExecuteShellCommand</a></li>
+<li><a href="pluggable/scm/gerrit/GerritSCMProtocol.html" title="enum in pluggable.scm.gerrit" target="classFrame">GerritSCMProtocol</a></li>
+<li><a href="pluggable/scm/gerrit/GerritSCMProvider.html" title="class in pluggable.scm.gerrit" target="classFrame">GerritSCMProvider</a></li>
+<li><a href="pluggable/scm/gerrit/GerritSCMProviderFactory.html" title="class in pluggable.scm.gerrit" target="classFrame">GerritSCMProviderFactory</a></li>
+<li><a href="pluggable/scm/helpers/HelperUtils.html" title="class in pluggable.scm.helpers" target="classFrame">HelperUtils</a></li>
```

Comment:
Thanks @SachinKSingh28 for the PR. This class was renamed to PropertyUtils. Please can we regenerate the docs from latest on feature/bitbucket-support?